src/PerfCounters.cc

```diff
@@ -32,23 +32,29 @@ namespace rr {
 #define PERF_COUNT_RR 0x72727272L
 
 static bool attributes_initialized;
-// pmu_bug_flags is an architecture dependent flags to determine
-// what bugs need to be checked.
-// Current, this is simply the uarch on x86 and unused on aarch64.
-static int pmu_bug_flags;
 // At some point we might support multiple kinds of ticks for the same CPU arch.
 // At that point this will need to become more complicated.
-static struct perf_event_attr ticks_attr;
-static struct perf_event_attr minus_ticks_attr;
-static struct perf_event_attr cycles_attr;
-static struct perf_event_attr llsc_fail_attr;
-static uint32_t pmu_flags;
-static uint32_t skid_size;
+struct perf_event_attrs {
+  // bug_flags is an architecture dependent flags to determine
+  // what bugs need to be checked.
+  // Current, this is simply the uarch on x86 and unused on aarch64.
+  int bug_flags{0};
+  perf_event_attr ticks{};
+  perf_event_attr minus_ticks{};
+  perf_event_attr cycles{};
+  perf_event_attr llsc_fail{};
+  uint32_t pmu_flags{0};
+  uint32_t skid_size{0};
```

Comment:
I'd prefer `= 0` for these initializers. We're not using brace-initializers elsewhere in rr (AFAIK) and brace-initializers don't give us anything particularly helpful here.